### PR TITLE
scheduler: updated PR scheduler for readability and portability

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to update the existing PR scheduler to improve
readability and portability.

**Changes proposed in this PR:**

* Version 2:
1. Removed duplicated statements.
2. Added debug message for a server administrator
3. Changed waiting time to avoid unexpected entrance into a run queue

* Version 1:
1. Updated statement to compare current queues and maximum queues.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

File: ci/taos/checker-pr-audit-async.sh [hunk 1]
```diff
@@ -233,8 +233,9 @@ popd
 
 # Pull-Request scheduler for Tizen build (gbs)
 # The default RUN Queue is declared in the configuration file
-while [ `ps aux | grep "sudo.*gbs build" | wc -l` -gt $RUN_QUEUE_PR_JOBS ]; do
-    WAITTIME=$(( ( RANDOM % 20 ) + 20 ))
+current_jobs=$(ps aux | grep "sudo.*gbs build" | wc -l)
+while [ $current_jobs -gt $RUN_QUEUE_PR_JOBS ]; do
```
Comment (ohsewon) [inline ci/taos/checker-pr-audit-async.sh]: current_jobs doesn't changed. so it can make infinite loop

Comment (leemgs) [reply ci/taos/checker-pr-audit-async.sh]: Updated. resubmitted.

**Comment by leemgs** [comment]
**Changes proposed in this PR:**

* Version 2:
1. Removed duplicated statements.
2. Added debug message for a server administrator
3. Changed waiting time to avoid unexpected entrance into a run queue

* Version 1:
1. Updated statement to compare current queues and maximum queues.

**Comment by leemgs** [comment]
The aging test is okay. This PR is ready to be merged.

**Pull request merged**